operations/tempo-mixin-compiled/dashboards/tempo-operational.json

```diff
@@ -5565,7 +5565,7 @@
      "type": "prometheus",
      "uid": "$ds"
     },
-    "definition": "label_values(tempo_build_info, namespace)",
+    "definition": "label_values(tempo_build_info{cluster=~'$cluster'}, namespace)",
```

Comment:
Do these need to be double quotes?

Reply:
tested in Grafana, it seems to work with single quotes.

Reply:
tested again, it works.